src/containers/StageMenu.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { actionCreators as menuActions } from '../ducks/modules/menu';
+import { actionCreators as stageActions } from '../ducks/modules/stage';
+import { stages, stage } from '../selectors/session';
+import { Menu } from '../components';
+
+const menuIsOpen = state => state.menu.menuIsOpen;
+
+/**
+  * Renders a Menu using stages to construct items in the menu
+  * @extends Component
+  */
+class StageMenu extends Component {
+  /**
+    * adds search term and list of matching stages to local state
+    */
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
I had thought of the search term as something more temporary, but I can move it to redux if that is a better place for it.